amida-tech/greyscale
# task/INBA-529-PO-Add Message Services (notify) call from Discussions and Tasks

Add ability for users to get internal messages as well as emails when they get flagged. 

To test:
- Go through steps to create project, assign task etc.
- Comment on a question and notify user.
- Log in as that user and make sure they get notification on Indaba and they also get an email.
